RetailCoder.VBE/UI/RubberduckUI.de.resx

```diff
@@ -899,7 +899,7 @@ Warnung: Alle eigenen Einstellungen gehen dabei verloren. Die Originaldatei wird
     <value>Versionsverwaltung</value>
   </data>
   <data name="SourceControl_AttributesFile" xml:space="preserve">
-    <value>Attributes File</value>
+    <value>'.gitattributes' Datei</value>
```

Comment:
What do you expect from the wording "Ignore File" and "Attribute File". If you are not familiar with git this does not really help.
What is the idea of "Ignore File" without knowing git I supose do not use the file.

May be it is a good Idea to add a tooltip to explain what is the meaning of ".gitignore" and ".gitattributes".